blocks/extra-large-promo-block/features/extra-large-promo/default.jsx

```diff
@@ -1,123 +1,245 @@
+			language: "en",
+			timeZone: "GMT",
+			dateTimeFormat: "LLLL d, yyyy 'at' K:m bbbb z",
+		},
+		fallbackImage,
+		locale,
+	} = getProperties(arcSite);
+	const {
+		imageOverrideURL,
+		imageRatio,
+		itemContentConfig,
+		lazyLoad,
+		playVideoInPlace,
+		showByline,
+		showDate,
+		showDescription,
+		showHeadline,
+		showImage,
+		showOverline,
+	} = customFields;
 
-const ExtraLargePromoItem = ({ customFields, arcSite }) => {
 	const content =
 		useContent({
-			source: customFields?.itemContentConfig?.contentService ?? null,
-			query: customFields?.itemContentConfig?.contentConfigValues
+			source: itemContentConfig?.contentService ?? null,
+			query: itemContentConfig?.contentConfigValues
 				? {
 						feature: "extra-large-promo",
-						...customFields.itemContentConfig.contentConfigValues,
+						...itemContentConfig.contentConfigValues,
 				  }
 				: null,
 			filter: `{
-      _id
-      credits {
-        by {
-          _id
-          name
-          url
-          type
-          additional_properties {
-            original {
-              byline
-            }
-          }
-        }
-      }
-      description {
-        basic
-      }
-      display_date
-      type
-      headlines {
-        basic
-      }
-      label {
-        basic {
-          display
-          url
-          text
-        }
-      }
-      owner {
-        sponsored
-      }
-      promo_items {
-        type
-        url
-        lead_art {
-          embed_html
-          type
-          promo_items {
-            basic {
-              type
-              url
-              resized_params {
-                800x600
-                800x533
-                800x450
-                600x450
-                600x400
-                600x338
-                400x300
-                400x267
-                400x225
-              }
-            }
-          }
-        }
-        basic {
-          type
-          url
-          resized_params {
-            800x600
-            800x533
-            800x450
-            600x450
-            600x400
-            600x338
-            400x300
-            400x267
-            400x225
-          }
-        }
-      }
-      website_url
-      embed_html
-      websites {
-        ${arcSite} {
-          website_url
-          website_section {
-            _id
-            name
-          }
-        }
-      }
-    }`,
+				_id
+				credits {
+					by {
+						_id
+						name
+						url
+						type
+						additional_properties {
+							original {
+								byline
+							}
+						}
+					}
+				}
+				description {
+					basic
+				}
+				display_date
+				type
+				headlines {
+					basic
+				}
+				label {
+					basic {
+						display
+						url
+						text
+					}
+				}
+				owner {
+					sponsored
+				}
+				promo_items {
+					type
+					url
+					lead_art {
+						embed_html
+						type
+						promo_items {
+							basic {
+								type
+								url
+								resized_params {
```

Comment:
Assuming that we'll be taking these out as a Resizer v2 task?

Reply:
That is my assumption as well